packages/smart-contracts/test/contracts/BatchConversionPayments.test.ts

```diff
@@ -386,56 +382,55 @@ describe('contract: BatchConversionPayments', async () => {
           },
         ],
         feeAddress,
-        { value: 1000 + 1 + 42 },
+        { value: 1000 + 1 + 42 }, // +42 in excess
       );
 
-      await checkETHBalances(
+      await expectETHBalanceDiffs(
         BigNumber.from(1000),
         BigNumber.from(1),
         BATCH_FEE,
-        beforeETHBalanceFrom,
-        beforeETHBalanceTo,
-        beforeETHBalanceFee,
+        initialFromETHBalance,
+        initialToETHBalance,
+        initialFeeETHBalance,
       );
     });
 
     it('make 1 ETH payment with 1-step conversion', async function () {
       // get Eth balances
-      const beforeETHBalanceTo = await provider.getBalance(to);
-      const beforeETHBalanceFee = await provider.getBalance(feeAddress);
-      const beforeETHBalanceFrom = await provider.getBalance(await fromSigner.getAddress());
+      const initialToETHBalance = await provider.getBalance(to);
+      const initialFeeETHBalance = await provider.getBalance(feeAddress);
+      const initialFromETHBalance = await provider.getBalance(await fromSigner.getAddress());
       tx = await batchConversionProxy.batchRouter(
         [
           {
-            paymentNetworkId: 4,
+            paymentNetworkId: PaymentTypes.BatchPaymentNetworkId.batchEthConversionPayments,
             conversionDetails: [ethConvDetail],
             cryptoDetails: emptyCryptoDetails,
           },
         ],
         feeAddress,
         {
-          value: (1000 + 1 + 42) * USD_ETH_RATE,
+          value: (1000 + 1 + 42) * USD_ETH_RATE, // +42 in excess
```

Comment:
Just FYI, in the other test, it's 42 ETH in excess, here it's 42 USD, but it tests the same thing so these tests should probably be consistent with each other.

Reply:
42 was a proxy for the batch fees amount: 1001 *10% which is about  +11 (I updated this)
Regarding this, we have to multiply the USD into ETH.
this will failed: (1000 + 1) * USD_ETH_RATE + 42